modules/installation-gcp-config-yaml.adoc

```diff
@@ -72,23 +70,23 @@ endif::[]
 platform:
   gcp:
     ProjectID: openshift-production <1>
-    region: us-central-1 <1>
+    region: us-central1-a <1>
```

Comment:
us-central1-a is a zone rather than a region. Please update it to us-central1. Thanks

Reply:
@shellyyang1989, thank you!